common/v2/services/Store/Network/helpers.ts

```diff
@@ -38,6 +43,11 @@ export const getNetworkById = (id: string): Network | undefined => {
   return networks.find((network: Network) => network.id === id);
 };
 
+export const getNetworkByDPath = (dPath: DPath): Network | undefined => {
+  const networks = getAllNetworks() || [];
+  return networks.find((network: Network) => network.dPaths.default ? network.dPaths.default.value === dPath.value : false);
+};
```

Comment:
This will return the first match. Since multiple networks share similar dPaths, this is close to a random network selection. Please explain.

Reply:
Once again, networks default dPath are not unique so they cannot be used to find a network.
e.g the Homestead and POA both share the same default path.

Reply:
Yeah, I'm aware. It has already been removed. Not sure why I hadn't marked it resolved.